### PR TITLE
✨ FEAT (project): export project models in index

### LINKED ISSUE #37: Inconsistent exports from module's index
## Description 

At the moment, the exports from [Project's index](https://github.com/niji-co/Fusion-Web/blob/main/src/pages/Project/index.ts) and [Project Row's index](https://github.com/niji-co/Fusion-Web/blob/main/src/pages/Project/Row/index.ts) are different

### Comparison

section | Project | Project Row
--- | --- | ---
view | ✅  | ✅
reducer | ✅  | ✅
selector | ✅  | ✅
model |   | ✅

Looking at the table, they are more or less the same with the exception of project's model.

### PR DESCRIPTION
**Pull request by lim-james**
Resolves #37

### CODE REVIEW

**Comment by sophiewongsp** [comment]
Build failed on commit #0ba5fec

**Review by sophiewongsp — APPROVED** [review]
Looks good :D

File: src/pages/Project/models/ProjectQueryMode.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+interface ProjectQueryMode {
+  title: string;
+}
+
+export default ProjectQueryMode;
```
Comment (sophiewongsp) [inline src/pages/Project/models/ProjectQueryMode.ts:5]: Rename to ProjectQueryModel

File: src/pages/Project/Project.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { RootState } from "services/store";
 
 import ProjectPreviewLayout from "./ProjectPreviewLayout";
 
-import ProjectQueryModel from "./models/ProjectQueryModel";
+import ProjectQueryModel from "./models/ProjectQueryMode";
```
Comment (sophiewongsp) [inline src/pages/Project/Project.tsx:9]: Revert back to ProjectQueryModel

File: src/pages/Project/models/ProjectQueryMode.ts (hunk 1 above)
Comment (lim-james) [reply src/pages/Project/models/ProjectQueryMode.ts]: Oops

**Pull request merged**